end-to-end-test/remote/specs/core/home.spec.js

```diff
@@ -21,11 +21,15 @@ describe('homepage', function() {
     });
 
     if (useExternalFrontend) {
-        it('it should show dev mode when testing', function() {
-            var devMode = $('.alert-warning');
-
-            devMode.waitForExist(10000);
-            assert(browser.getText('.alert-warning').indexOf('dev mode') > 0);
+        it('window.frontendConfig.frontendUrl should point to localhost 3000 when testing', function() {
+            // We no longer check whether the dev mode banner exits.
+            // The banner is hidden in e2etests.scss
+            assert.equal(
+                browser.execute(function() {
+                    return window.frontendConfig.frontendUrl;
+                }).value,
+                '//localhost:3000/'
+            );
```

Comment:
@inodb @alisman here is the updated test. Let me know whether I'm doing it right. Thank you.

Reply:
looks right to me

Reply:
looks good to me